starter-code/src/contactsList.js

```diff
@@ -1,4 +1,4 @@
-[
+const contacts = [
```

Comment:
Following the assignment, this data was supposed to be stored in a .json file and not in a .js file. 

But, I like the your approach on seperating the logic for selecting the first five apart from your components.js. This approach can come in very handy when we will be working with lifecycle methods :)